data-prepper-plugins/sns-sink/src/test/java/org/opensearch/dataprepper/plugins/sink/SNSSinkServiceTest.java

```diff
@@ -0,0 +1,147 @@
+package org.opensearch.dataprepper.plugins.sink;
```

Comment:
This file is missing the copyright header

Reply:
Thanks @graytaylor0 for your feedback. Done.